packages/sw-runtime-caching/build.js

```diff
@@ -13,41 +13,18 @@
  limitations under the License.
 */
 
-const rollupBabel = require('rollup-plugin-babel');
-const path = require('path');
-const {buildJSBundle} = require('../../build-utils');
 const pkg = require('./package.json');
+const rollupBabel = require('rollup-plugin-babel');
+const {buildJSBundle, generateBuildConfigs} = require('../../build-utils');
+
+const plugins = [rollupBabel({
```

Comment:
I was initially hesitant to add things to the build process that weren't needed for a given library, but I just did a quick comparison of before/after when using all the plugins, and there's either a negligible or no change in build time and output file size.

So I'll just standardize.